### DISCUSSION TITLE
once run, it says downloading language model

### DISCUSSION THREAD

**Discussion by zhang402115** [BLKSerene/Wordless#30] [General]
But downloading process can not be finished, where shall i put the netdisk downloaded language model?

**Comment by BLKSerene** [comment]
Connection to Gihub in China is unstable so it it recommended to use a proxy. If you encounter network error, there is nothing you can do except retrying.

For `Wordless 3.3.0`, download the zip file for spaCy models (https://pan.baidu.com/s/1--ZzABrDQBZlZagWlVQMbg?pwd=wdls#list/path=%2FWordless%2Fmodels%2FspaCy&parentPath=%2F) and extract all files. Put the extracted files under the root directory of Wordless on Windows and Linux, or under `Wordless.app/Contents/MacOS` on macOS.
If language models are missing from the Baidu Netdisk, let me know the language of your corpus and I'll upload more.
